.github/workflows/build.yml

```diff
@@ -10,24 +10,21 @@ jobs:
           - ruby: 3.3
             gemfile: Gemfile
           - ruby: 3.2
-            gemfile: Gemfile
+            gemfile: gemfiles/rails71.gemfile
           - ruby: 3.1
             gemfile: gemfiles/rails70.gemfile
           - ruby: "3.0"
             gemfile: gemfiles/rails61.gemfile
-          - ruby: 2.7
-            gemfile: gemfiles/rails60.gemfile
-          - ruby: 2.6
-            gemfile: gemfiles/rails52.gemfile
+          - ruby: 3.3
+            gemfile: gemfiles/mongoid9.gemfile
+            mongodb: true
           - ruby: 3.1
             gemfile: gemfiles/mongoid8.gemfile
             mongodb: true
-          - ruby: 2.7
-            gemfile: gemfiles/mongoid7.gemfile
-            mongodb: true
-          - ruby: 2.6
-            gemfile: gemfiles/mongoid6.gemfile
+          - ruby: 3.1
+            gemfile: gemfiles/mongoid9.gemfile
```

Comment:
This test case was not in the upstream, but I added it since this is the point of the upgrade.